bot/exts/halloween/spooknamerate.py

```diff
@@ -0,0 +1,325 @@
+                return
+
+        await ctx.send("But you don't have an entry... :eyes: Type `spooknamerate add your entry`")
+
+    @tasks.loop(hours=24.0)
+    async def announce_word(self) -> None:
+        """Announces the name needed to spookify every 24 hours and the winner of the previous game."""
+        if not datetime.utcnow().month == Month.OCTOBER:
+            return
+
+        channel = await self.get_channel()
+
+        if self.first_time:
+            await channel.send("Okkey... Welcome to **Spook Name Rate**! It's a relatively simple game.\n"
+                               "Everyday, a random name will be sent in `#seasonalbot-commands` "
+                               "and you need to try and spookify it!\nRegister your word using "
+                               f"`{self.bot.command_prefix}spooknamerate add spookified_name`")
+
+            self.first_time = False
+
+        else:  # otherwise,
+            if self.messages:
+                await channel.send(embed=await self.get_responses_list(final=True))
+                self.poll = True
+                await asyncio.sleep(2 * 60 * 60)
+
+            logger.info('Calculating score')
+            for message_id in self.messages:
+                msg = await channel.fetch_message(message_id)  # fetch the message
+                score = 0
+                reactions = msg.reactions  # get the reactions
+                for reaction in reactions:
+                    reaction_value = self.emojis_val.get(reaction.emoji, 0)  # get the value of the emoji else 0
+                    score += reaction_value * (reaction.count - 1)  # multiply by the num of reactions
+                    # subtract one, since one reaction was done by the bot
+
+                logger.debug(f'{msg.author} got a score of {score}')
+                self.messages[message_id]['score'] = score
+
+            winner_messages = sorted(self.messages.items(), key=lambda x: x[1]['score'], reverse=True)
+
+            # Sort the winner messages
+            winners = []
+            for i, winner in enumerate(winner_messages):
+                winners.append(winner)
+                if len(winner_messages) > i + 1:
+                    if winner_messages[i + 1][1]['score'] != winner[1]['score']:
+                        break
+                elif len(winner_messages) == (i + 1) + 1:  # The next element is the last element
+                    if winner_messages[i + 1][1]['score'] != winner[1]['score']:
+                        break
+
+            # one iteration is complete
+            await channel.send("Today's Spook Name Rate Game ends now, and the winner(s) is(are)...")
+
+            async with channel.typing():
+                await asyncio.sleep(1)  # give the drum roll feel
+
+                if winners:  # make sure there is a winner
+
+                    if len(winners) > 1:
+                        await channel.send(" and ".join([win[1]['author'].mention for win in winners]) + " !")
```

Comment:
```suggestion
                        await channel.send(" and ".join(win[1]['author'].mention for win in winners) + " !")
```
A generator works just as well without creating a list for it